### PR TITLE
EvolveAllPkm Management - Evolve all above certain IV value

### PR DESCRIPTION
**Pull request by skupfer**
BUT NOW. PLEASE

EDIT:

Evolve all pokemon can be true and/or evolveallaboveiv
there evolveaboveivvalue comes into play ...

EDIT2: If you like an PR, don't close it immediately please. Let us try to resolve the merge conflicts first. Last time everything got a little bit messed up somehow...

### CODE REVIEW

**Comment by skupfer** [comment]
If it doesn't work this time, I am sorry. Maybe someone else is able to be fast enough with this git push merge stuff to be ahead :D

**Comment by solenrus** [comment]
bruh, gl

File: PokemonGo.RocketAPI.Console/App.config [hunk 1]
```diff
@@ -61,6 +61,14 @@
...
         <value>1</value>
       </setting>
+      <setting name="PrioritizeIVOverCP" serializeAs="String">
+        <value>False</value>
+      </setting>
+      <setting name="EvolveAllPokemonAboveIV" serializeAs="String">
+        <value>False</value>
+      </setting>
+      <setting name="EvolveAboveIVValue" serializeAs="String">
+        <value>0</value>
```
Comment (snotyak) [inline PokemonGo.RocketAPI.Console/App.config]: You lost your closing tag for this. Probably from a bad merge.

File: PokemonGo.RocketAPI.Console/UserSettings.settings [hunk 2]
```diff
@@ -47,6 +47,11 @@
     <Setting Name="PrioritizeIVOverCP" Type="System.Boolean" Scope="User">
       <Value Profile="(Default)">False</Value>
     </Setting>
+    <Setting Name="EvolveAllPokemonAboveIV" Type="System.Boolean" Scope="User">
+      <Value Profile="(Default)">False</Value>
+    </Setting>
+    <Setting Name="EvolveAboveIVValue" Type="System.Single" Scope="User">
+      <Value Profile="(Default)">0</Value>
```
Comment (snotyak) [inline PokemonGo.RocketAPI.Console/UserSettings.settings]: You lost your closing tag for this. Probably from a bad merge.

**Comment by NecronomiconCoding** [comment]
fix conflicts

**Comment by skupfer** [comment]
No function then I guess. This was the third try

**Pull request closed without merging**